src/tabs.ts

```diff
@@ -87,30 +87,34 @@ export class Tabs extends Component<TabsOptions> {
    * @param els HTML elements.
    * @param options Component options.
    */
-  static init(els: HTMLElement | InitElements<MElement>, options: Partial<TabsOptions> = {}): Tabs | Tabs[] {
+  static init(
+    els: HTMLElement | InitElements<MElement>,
+    options: Partial<TabsOptions> = {}
+  ): Tabs | Tabs[] {
     return super.init(els, options, Tabs);
   }
 
   static getInstance(el: HTMLElement): Tabs {
-    return (el as any).M_Tabs;
+    return el['M_Tabs'];
   }
 
   destroy() {
     this._removeEventHandlers();
     this._indicator.parentNode.removeChild(this._indicator);
     if (this.options.swipeable) {
       this._teardownSwipeableTabs();
-    }
-    else {
+    } else {
       this._teardownNormalTabs();
     }
-    (this.el as any).M_Tabs = undefined;
+    this.el['M_Tabs'] = undefined;
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition